packages/loaders/src/LoaderResource.ts

```diff
@@ -13,14 +13,16 @@ import { Spritesheet } from '@pixi/spritesheet';
 export interface IResourceMetadata extends Resource.IMetadata {
     imageMetadata?: any;
 }
-export class LoaderResource extends Resource
+export interface ILoaderResource extends Resource
```

Comment:
I changed `class ILoaderResource` into interface `ILoaderResource` bacause we don't construct class directly, and when we call `resource instanceof ResourceLoader` it is _false_, because **resource** instance of **Resource**
We not need realy class for typesafe,we can use interface.